modules/bitcoind.nix

```diff
@@ -266,7 +267,9 @@ in {
         readOnly = true;
         type = types.package;
         default = pkgs.writeScriptBin "bitcoin-cli" ''
-          exec ${cfg.package}/bin/bitcoin-cli -datadir='${cfg.dataDir}' "$@"
+          exec ${cfg.package}/bin/bitcoin-cli \
+          -rpcuser=${cfg.rpc.users.privileged.name} -rpcpassword=$(cat ${secretsDir}/bitcoin-rpcpassword-privileged) \
```

Comment:
Perhaps better to leave the rpcuser and rpcpassword in the bitcoin.conf so we don't have to specify them here

Reply:
`-rpcpassword` being entered from command line is actually insecure. `bitcoin-cli` has an option `-stdinrpcpass` to solve this problem - just read it from that file.

Reply:
Fixed in fb94d19
I think it's even better to use the config file than `-stdinrpcpass`

Reply:
BTW, this is just defense in depth since we already forbid users from viewing other's commands with hidepid